### DISCUSSION TITLE
Component Based Organization

### DISCUSSION THREAD

**Discussion by patrickbjohnson** [uicrooks/shopify-foundation-theme#66] [Ideas]
The current set for Vue Components is based on their `render` or `renderless` capabilities. 
Has there been any though to allowing for allowing a component based configuration? What I've seen in some cases is a `components` directory with children folders representing the `js` files for each component. I've also seen some projects in which all necessary items are in the component's folder `js`, `css`, `assets`, etc.

**Comment by sergejcodes** [comment]
I'm not sure if I'm understanding the description correctly, but the component directory structure is flexible. By default, it's grouped into `render` and `renderless` it can be anything though.
